.golangci.yml

```
@@ -1,3 +1,4 @@
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)